src/tower_arduino_ide/linac.cpp

```diff
@@ -119,8 +125,9 @@ uint8_t linac_retract()
     if(current_linac_state == LINAC_RETRACTED)
         return true;
 
-    digitalWrite(PIN_LINACTUATOR_SELECT, LOW); 
-    digitalWrite(PIN_LINACTUATOR_POWER, HIGH);
+    // digitalWrite(PIN_LINACTUATOR_SELECT, LOW); 
+    // digitalWrite(PIN_LINACTUATOR_POWER, HIGH);
+    relay_bus_write(LINAC_ADDR, LINAC_PWR_HIGH + LINAC_SEL_LOW);
```

Comment:
This is kinda weird... The way your data is formatted isn't very clear and it's hard to sanity check/extend later.

Reply:
Maybe I should add a comment, but the data is formatted to be use one-hot encoding to be later decoded on the PIC side